src/bridge/cockpitdbusjson.c

```diff
@@ -122,6 +122,30 @@ check_type (JsonNode *node,
   return TRUE;
 }
 
+static gboolean
+check_int_type (JsonNode *node,
+                const char *type_name,
```

Comment:
I didn't do that because D-Bus' type strings are hard to remember, especially the integers. But I grant you that when dealing with D-Bus, you need to know them anyway and introducing different ones is weird. Changed in the update.